### PR TITLE
feature/model update v3

### PR DESCRIPTION
**Pull request by fdiehr**
# PR Context
We need to update our metadata model due to new insights deriving from a) consulting more standards and schemas specific to the modelling of health data, like upcoming profile "HealthDCAT-AP" in relation to the development of the EHDS (European Health Data Space).
Further, we introduce a new entity called "bibliographic reference". This new entity allows us to describe bibliographic references, e.g. publications (books, book sections, journal articles). With that, we added a new core entity to our metadata catalogue, which provides valuable information about RKI's research.  

# Added
- Entity _bibliographic reference_
- Vocabulary _bibliographic-reference-type_: a list of different publication types. 
- Vocabulary _resource-creation-method_: we want to have a better distinction between the usage of _resourceTypeGeneral_ as a generic type for resources and the context/method of how the resources were created.
- Vocabulary _personal-data_: we want to be able to say, if a resource contains personal data or not.
- Added JSON Schema property _description_ to all entities and properties, since we delete the specification of the model (docs/index.md)

**resource**
- Property _conformsTo_: Providing information about standards used with the resource.
- Property _doi_: For published resources.
- Property _hasLegalBasis_: Information about the legal basis on which the data was processed; inspired by HealthDCAT-AP draft and [Data Privacy Vocabulary](https://w3c.github.io/dpv/dpv/#hasLegalBasis)
- Property _hasPersonalData_: Information if the resource contains personal data or not ;inspired by HealthDCAT-AP draft and [Data Privacy Vocabulary](https://w3c.github.io/dpv/dpv/#hasPersonalData)
- Property _maxTypicalAge_ and _minTypicalAge_: Information about the age of the researched population; inspired by HealthDCAT-AP draft
- Property _populationCoverage_: Information about the researched population; inspired by HealthDCAT-AP draft
- Property _resourceCreationMethod_: Information about the context or method used in order to create the resource

# Changes
all entities
- Property _identifierInPrimarySource_ has now a "minLength" and "maxLength" and has a pattern to restrict newlines

**variable**
- Property _dataType_ does not expect a vocabulary item anymore, but a string

**activity**
- Property _publication_ is now expecting the identifier of _bibliographic-reference_ item

**resource**
- Property _publication_ is now expecting the identifier of a *bibliographic reference* item
- Property _loincId_: new pattern

**distribution**
- Properties _accessURL_ and _downloadURL_ now have the type "array", because there can be multiple repositories for accessing the distribution (e.g. GitHub, Zenodo).
- Deleted some properties: We want only to have properties that are relevant for the distribution's access. Hence, all properties concerning actors involved in the creation of the resource were deleted.

**vocabularies**
- _theme_: It had too many concepts and was not suitable for our purposes. We decided to delete most of the concepts and keep only the main topics of RKI.
- _activity_: Deleted concepts due to discussions: We want to focus on the distinction between third-party funded projects and internal projects.
- _resource-type-general_: Deleted and added concepts due to discussions: We have consulted other standards and schemas (e.g. NFDI4Health, DataCite) to find more suitable resource types; although we didn't find a perfect fit for the vocabulary, we found a few useful concepts and added some of them.

**date properties** 
- Defined new patterns for dates: `YearMonthDay`, `YearMonth`, `Year`. Delete `"format": "date"` and `"format": "datetime"` to ensure validation. 

**index.md**
Deleted all information that were relevant to the "specification" of the model. We decided, that the maintenance of the specification is not worth its benefit (mainly interoperability with standards). In the future, it is more feasible and fitting to our use case to maintain mappings to the portals we will contribute to. 
But we want to keep the index file, because we may add some more description on the schema on a more general level.

**README.md**
- Transferred contact and publisher information from _index.md_ to README

# Removed
- Vocabulary _data-type.json_ because we decided against having a controlled vocabulary for data types.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,15 @@ find research data from the RKI.
...
 
+**Contact** \
+For more information, please feel free to email us at [mex@rki.de](mailto:mex@rki.de).
+
+### Publisher of this document
+**Robert Koch-Institut** \
+Nordufer 20 \
+13353 Berlin \
+Germany
+
```
Comment (cutoffthetop) [inline README.md:42]: if you want this in here, it should go into the template to be propagated into all repos https://github.com/robert-koch-institut/mex-template/blob/0.3.0/mex-%7B%7B%20cookiecutter.project_name%20%7D%7D/README.md?plain=1#L38

File: mex/model/entities/bibliographic-resource.json [hunk 2]
```diff
@@ -0,0 +1,416 @@
...
+            "type": "array"
+        },
+        "doi": {
+            "description": "The Digital Object Identifier (DOI) of the publication.",
+            "examples": [
+                "https://doi.org/10.1007/978-1-0716-2441-8_7",
+                "https://doi.org/10.2807/1560-7917.ES.2022.27.46.2200849",
+                "https://doi.org/10.3389/fmicb.2022.868887",
+                "http://dx.doi.org/10.25646/5147",
+                "https://doi.org/10.1016/j.vaccine.2022.11.065."
```
Comment (cutoffthetop) [inline mex/model/entities/bibliographic-resource.json:102]:
```suggestion
                "https://doi.org/10.1016/j.vaccine.2022.11.065"
```

File: README.md (hunk 1 above)
Comment (fdiehr) [reply README.md:42]: I think the information would be nice to have, and maybe also good to have the official RKI address present as well (kind of having an "impressum"). what do you think? if you agrre, then I can put this into mex-template.

File: mex/model/entities/bibliographic-resource.json [hunk 3]
```diff
@@ -0,0 +1,416 @@
...
+            "examples": [
+                "1",
+                "45-67",
+                "45 - 67",
+                "II",
+                "XI",
+                "10i"
+            ],
+            "type": "string"
+        },
```
Comment (cutoffthetop) [inline mex/model/entities/bibliographic-resource.json:296]:
```suggestion
        "pages": {
            "anyOf": [
                {
                    "examples": [
                        "1",
                        "45-67",
                        "45 - 67",
                        "II",
                        "XI",
                        "10i"
                    ],
                    "type": "string"
                },
                {
                    "type": "null"
                }
            ],
            "default": null,
            "description": "The range of pages or a single page."
        },
```

File: mex/model/entities/bibliographic-resource.json [hunk 4]
```diff
@@ -0,0 +1,416 @@
...
+        "publicationYear": {
+            "describing": "The year in which the publication was issued.",
+            "examples": [
+                "2011",
+                "1982",
+                "1810"
+            ],
+            "pattern": "^(?:\\d{4})$",
```
Comment (cutoffthetop) [inline mex/model/entities/bibliographic-resource.json:304]: this introduces a new timestamp format. up unitl now, we only had `YearMonthDayTime`, `YearMonthDay` and `YearMonth`. the latter has a required year and an optional month, but we cannot enforce that _just_  a year is given.

tldr; we have to convert each string with pattern `^(?:\\d{4}|(?:\\d{4}-(?:0[1-9]|1[0-2])))$` into two separate entries within their respective `anyOf`-lists. one for `YearMonth` (both required) and one for just `Year`

File: mex/model/entities/bibliographic-resource.json [hunk 5]
```diff
@@ -0,0 +1,416 @@
...
+            "sameAs": "http://datacite.org/schema/kernel-4/publicationYear",
+            "type": "string"
```
Comment (cutoffthetop) [inline mex/model/entities/bibliographic-resource.json:306]: also, if this is optional it needs the `anyOf`/`type: null`-treatment

File: mex/model/entities/bibliographic-resource.json [hunk 6]
```diff
@@ -0,0 +1,416 @@
...
+        },
+        "publisher": {
+            "description": "An entity responsible for making the publication available ([DCT, 2020-01-20](http://dublincore.org/specifications/dublin-core/dcmi-terms/2020-01-20/)).",
```
Comment (cutoffthetop) [inline mex/model/entities/bibliographic-resource.json:309]:
```suggestion
            "description": "An entity responsible for making the publication available ([DCT, 2020-01-20](http://dublincore.org/specifications/dublin-core/dcmi-terms/2020-01-20/)).",
            "default": [],
```

File: mex/model/entities/bibliographic-resource.json [hunk 7]
```diff
@@ -0,0 +1,416 @@
...
+                },
+                {
+                    "type": "null"
+                }
+            ],
+            "default": null,
+            "description": "The handle of the publication in the repository, where the publication is stored."
+        },
+        "section": {
+            "description": "The name of the chapter of the publication, the book section belongs to.",
```
Comment (cutoffthetop) [inline mex/model/entities/bibliographic-resource.json:329]: anyOf/null again

File: mex/model/entities/bibliographic-resource.json [hunk 8]
```diff
@@ -0,0 +1,416 @@
...
+                "A Section About Public Health",
+                "Chapter XII: The History of Public Health",
+                "12",
+                "A",
+                "B."
+            ],
+            "type": "string"
+        },
+        "subtitle": {
+            "description": "The subtitle of the publication.",
```
Comment (cutoffthetop) [inline mex/model/entities/bibliographic-resource.json:341]: `"default": [],`

File: mex/model/entities/bibliographic-resource.json [hunk 9]
```diff
@@ -0,0 +1,416 @@
...
+                "http://purl.org/dc/terms/title"
+            ],
+            "type": "array"
+        },
+        "titleOfBook": {
+            "description": "The title of the book in which the book section is published.",
+            "items": {
+                "$ref": "/schema/fields/text"
+            },
+            "minItems": 1,
```
Comment (cutoffthetop) [inline mex/model/entities/bibliographic-resource.json:366]: if it's minItems=1, it should go into the `required` list at the bottom

File: mex/model/entities/bibliographic-resource.json [hunk 10]
```diff
@@ -0,0 +1,416 @@
...
+                "http://purl.org/dc/terms/title"
+            ],
+            "type": "array"
+        },
+        "titleOfSeries": {
+            "description": "The title of the book series, the book belongs to.",
+            "items": {
+                "$ref": "/schema/fields/text"
+            },
+            "minItems": 1,
```
Comment (cutoffthetop) [inline mex/model/entities/bibliographic-resource.json:377]: same, should go into `required` then

File: mex/model/entities/bibliographic-resource.json [hunk 11]
```diff
@@ -0,0 +1,416 @@
...
+            "type": "array"
+        },
+        "volume": {
+            "description": "The volume of the periodical.",
+            "examples": [
+                "24",
+                "34",
+                "1"
+            ],
+            "type": "integer"
```
Comment (cutoffthetop) [inline mex/model/entities/bibliographic-resource.json:390]: same as with `issue`: are we **sure** this will stay numerical?

and, regardless: needs `anyOf/null, default=null` treatment

File: mex/model/entities/bibliographic-resource.json [hunk 12]
```diff
@@ -0,0 +1,416 @@
...
+        },
+        "volumeOfSeries": {
```
Comment (cutoffthetop) [inline mex/model/entities/bibliographic-resource.json:392]: see volume/issue

**Review by cutoffthetop — CHANGES_REQUESTED** [review]

File: mex/model/entities/bibliographic-resource.json [hunk 13]
```diff
@@ -0,0 +1,416 @@
...
+                    "1467-9442"
+                ],
+                "type": "string"
+            },
+            "sameAs": [
+                "http://datacite.org/schema/kernel-4/alternateIdentifier"
+            ],
+            "type": "array"
+        },
+        "issue": {
```
[the remark this answers is not in the document]
Comment (fdiehr) [reply mex/model/entities/bibliographic-resource.json:187]: good point. but I think for "issue" it is not very likely, since it it only relevant for journals and they stick to numbering their issues. but for "volume" it could be possible, since this can be also used for books ("Sammelband"). but maybe it is just safer to change both properties to be valid having also string values?

File: mex/model/entities/bibliographic-resource.json [hunk 14]
```diff
@@ -0,0 +1,416 @@
...
+            "description": "Date of formal issuance of the publication ([DCT, 2020-01-20](http://dublincore.org/specifications/dublin-core/dcmi-terms/2020-01-20/)).",
+            "sameAs": [
+                "http://purl.org/dc/terms/created"
+            ]
+        },
+        "journal": {
+            "default": [],
+            "description": "The periodical in which the article was published.",
+            "items": {
+                "$ref": "/schema/fields/text"
```
[the remark this answers is not in the document]
Comment (fdiehr) [reply mex/model/entities/bibliographic-resource.json:253]: hmm, but even, when we only have a value for one language, I thought we would like to have the language information, e.g. that "Journal of Health Monitoring" is English?

File: mex/model/entities/bibliographic-resource.json (hunk 4 above)
Comment (fdiehr) [reply mex/model/entities/bibliographic-resource.json:321]: okay, I wasn't aware of that, sorry. for publications, it is common to only give the year, so we cannot change that. but your suggestion sound good to me! I will change it accordingly.

File: mex/model/entities/bibliographic-resource.json (hunk 14 above)
[the remark this answers is not in the document]
Comment (cutoffthetop) [reply mex/model/entities/bibliographic-resource.json:253]: okay, yeah that's fair

File: mex/model/entities/bibliographic-resource.json (hunk 4 above)
Comment (fdiehr) [reply mex/model/entities/bibliographic-resource.json:342]: like that?
`        "publicationYear": {
            "anyOf": [
                {
                    "examples": [
                        "2011",
                        "1982",
                        "1810"
                    ],
                    "pattern": "^(?:\\d{4})$",
                    "type": "string"
                },
                {
                    "examples": [
                        "1903-01",
                        "2019-03"
                    ],
                    "pattern": "^(?:\\d{4}|(?:\\d{4}-(?:0[1-9]|1[0-2])))$",
                    "type": "string"
                }
            ],
            "description": "The year in which the publication was issued.",
            "sameAs": "http://datacite.org/schema/kernel-4/publicationYear",
            "type": "array"
        },`

> convert each string with pattern ^(?:\\d{4}|(?:\\d{4}-(?:0[1-9]|1[0-2])))$
do you mean for all properties in all entities?

File: mex/model/entities/activity.json [hunk 15]
```diff
@@ -179,15 +196,17 @@
         },
         "publication": {
             "default": [],
+            "description": "A publication related to the activity.",
             "items": {
-                "$ref": "/schema/fields/link"
+                "$ref": "/schema/entities/bibliographic-reference#/identifier"
```
Comment (cutoffthetop) [inline mex/model/entities/activity.json:201]:
```suggestion
                "$ref": "/schema/entities/bibliographic-resource#/identifier"
```

File: mex/model/entities/resource.json [hunk 16]
```diff
@@ -338,14 +430,33 @@
...
+                "$ref": "/schema/fields/text"
+            },
+            "type": "array"
+        },
         "publication": {
             "default": [],
+            "description": "A publication that deals with the research, analysis, observation or processing of the object that was subject to the resource, e.g. a research paper.",
             "items": {
-                "$ref": "/schema/fields/link"
+                "$ref": "/schema/entities/bibliographic-reference#/identifier"
```
Comment (cutoffthetop) [inline mex/model/entities/resource.json:459]:
```suggestion
                "$ref": "/schema/entities/bibliographic-resource#/identifier"
```

File: mex/model/entities/bibliographic-resource.json (hunk 4 above)
Comment (fdiehr) [reply mex/model/entities/bibliographic-resource.json:342]:
```suggestion
                    "publicationYear": {
            "anyOf": [
                {
                    "examples": [
                        "2011",
                        "1982",
                        "1810"
                    ],
                    "pattern": "^(?:\\d{4})$",
                    "type": "string"
                },
                {
                    "examples": [
                        "1903-01",
                        "2019-03"
                    ],
                    "pattern": "^(?:\\d{4}|(?:\\d{4}-(?:0[1-9]|1[0-2])))$",
                    "type": "string"
                }
            ],
            "description": "The year in which the publication was issued.",
            "sameAs": "http://datacite.org/schema/kernel-4/publicationYear",
            "type": "array"
        },
```

Comment (fdiehr) [reply mex/model/entities/bibliographic-resource.json:342]: like that?

`        "publicationYear": {
            "anyOf": [
                {
                    "examples": [
                        "2011",
                        "1982",
                        "1810"
                    ],
                    "pattern": "^(?:\\d{4})$",
                    "type": "string"
                },
                {
                    "examples": [
                        "1903-01",
                        "2019-03"
                    ],
                    "pattern": "^(?:\\d{4}|(?:\\d{4}-(?:0[1-9]|1[0-2])))$",
                    "type": "string"
                }
            ],
            "description": "The year in which the publication was issued.",
            "sameAs": "http://datacite.org/schema/kernel-4/publicationYear",
            "type": "array"
        },
`

File: mex/model/entities/distribution.json [hunk 17]
```diff
@@ -22,96 +24,69 @@
...
-            "subPropertyOf": [
-                "http://www.w3.org/ns/dcat#accessURL"
-            ]
-        },
-        "author": {
             "default": [],
+            "description": "A URL of the resource that gives access to a distribution of the dataset. E.g. landing page, feed, SPARQL endpoint ([DCAT, 2020-02-04](https://www.w3.org/TR/2020/REC-vocab-dcat-2-20200204/)).",
             "items": {
-                "$ref": "/schema/entities/person#/identifier"
+                "anyOf": [
```
Comment (cutoffthetop) [inline mex/model/entities/distribution.json:36]: inside of lists, we don't want to support `null` values. so this `anyOf` should be scratched, we just need `items: {ref: link}`.

same goes `downloadURL`

File: mex/model/entities/resource.json [hunk 18]
```diff
@@ -186,41 +217,70 @@
         },
         "hadPrimarySource": {
             "$ref": "/schema/entities/primary-source#/identifier",
+            "description": "A primary source for a topic refers to something produced by some agent with direct experience and knowledge about the topic, at the time of the topic's study, without benefit from hindsight. Because of the directness of primary sources, they 'speak for themselves' in ways that cannot be captured through the filter of secondary sources. As such, it is important for secondary sources to reference those primary sources from which they were derived, so that their reliability can be investigated. A primary source relation is a particular case of derivation of secondary materials from their primary sources. It is recognized that the determination of primary sources can be up to interpretation, and should be done according to conventions accepted within the application's domain ([PROV-O, 2013-04-30 ](http://www.w3.org/TR/2013/REC-prov-o-20130430/)).",
             "sameAs": [
                 "http://www.w3.org/ns/prov#hadPrimarySource"
             ]
         },
+        "hasLegalBasis": {
+            "$ref": "/schema/fields/text",
```
Comment (cutoffthetop) [inline mex/model/entities/resource.json:226]: should either be in required or set default null and allow anyOf/null

File: mex/model/entities/bibliographic-resource.json (hunk 4 above)
Comment (fdiehr) [reply mex/model/entities/bibliographic-resource.json]: can you please take a look, if I applied the changes correctly in `publicationYear` ?

File: mex/model/entities/resource.json [hunk 19]
```diff
@@ -186,41 +217,70 @@
...
+            "description": "The legal basis used to justify processing of personal data. Legal basis (plural: legal bases) are defined by legislations and regulations, whose applicability is usually restricted to specific jurisdictions which can be represented using dpv:hasJurisdiction or dpv:hasLaw. Legal basis can be used without such declarations, e.g. 'Consent', however their interpretation will require association with a law, e.g. 'EU GDPR'.",
+            "sameAs": [
+                "https://w3id.org/dpv#hasLegalBasis"
+            ]
+        },
+        "hasPersonalData": {
+            "default": [],
```
Comment (cutoffthetop) [inline mex/model/entities/resource.json:233]: why a list? since the semantics are basically boolean

File: mex/model/entities/bibliographic-resource.json (hunk 9 above)
Comment (fdiehr) [reply mex/model/entities/bibliographic-resource.json:434]: right, but I did it wrong in the first place. `titleOfBook and `titleOfSeries` must be optional, so I removed the `minItems` for both. 
the rule should be: if `titleOfBook or `titleOfSeries` are given, then it is required. but I guess that is not possible.

File: mex/model/entities/resource.json [hunk 20]
```diff
@@ -267,19 +330,37 @@
...
+                    "https://loinc.org/95209-3",
+                    "https://loinc.org/LA26211-5"
+                ],
+                "format": "uri",
+                "pattern": "^https://loinc.org/([a-zA-z]*)|(([0-9]{5}-[0-9]*))$",
                 "type": "string"
             },
             "type": "array"
         },
+        "maxTypicalAge": {
```
Comment (cutoffthetop) [inline mex/model/entities/resource.json:352]: required or anyOf/null

File: mex/model/entities/resource.json [hunk 21]
```diff
@@ -307,6 +390,15 @@
             ],
             "type": "array"
         },
+        "minTypicalAge": {
```
Comment (cutoffthetop) [inline mex/model/entities/resource.json:393]: required or null

File: mex/model/entities/bibliographic-resource.json (hunk 4 above)
Comment (cutoffthetop) [reply mex/model/entities/bibliographic-resource.json]: > do you mean for all properties in all entities?

yes, would be more consistent

Comment (cutoffthetop) [reply mex/model/entities/bibliographic-resource.json]: actually, our patterns were a bit inconsistent. i fixed them:
```json
{
    "year_month_day_time": "^\\d{4}-(?:0[1-9]|1[0-2])-(?:0[1-9]|[12][0-9]|3[01])T(?:[0-1][0-9]|2[0-3]):[0-5][0-9]:[0-5][0-9]Z$",
    "year_month_day": "^\\d{4}-(?:0[1-9]|1[0-2])-(?:0[1-9]|[12][0-9]|3[01])$",
    "year_month": "\\d{4}-(?:0[1-9]|1[0-2])$",
    "year": "^\\d{4}$"
}
```

File: mex/model/entities/bibliographic-resource.json [hunk 22]
```diff
@@ -0,0 +1,515 @@
...
+                        "1903-01",
+                        "2019-03"
+                    ],
+                    "pattern": "^(?:\\d{4}|(?:\\d{4}-(?:0[1-9]|1[0-2])))$",
+                    "type": "string"
+                },
+                {
+                    "type": "null"
+                }
+            ],
```
[the remark this answers is not in the document]
Comment (fdiehr) [reply mex/model/entities/bibliographic-resource.json:360]: right. but then I think I didn't fully understand your initial comment. did you mean to add the year pattern to the other date properties?

File: mex/model/vocabularies/bibliographic-resource-type.json [hunk 23]
```diff
@@ -0,0 +1,265 @@
...
+        "prefLabel": {
+            "de": "Buch",
+            "en": "Book"
+        }
+    },
+    {
+        "altLabel": {
+            "en": [
+                "Book section"
+            ]
```
Comment (cutoffthetop) [inline mex/model/vocabularies/bibliographic-resource-type.json:27]:
```suggestion
            "en": "Book section"
```

this can't be a list

Comment (cutoffthetop) [reply mex/model/vocabularies/bibliographic-resource-type.json:27]: also applies to the altLabels below. (maybe we should also have a schema for the vocabs 🤔)

File: mex/model/entities/bibliographic-resource.json [hunk 24]
```diff
@@ -0,0 +1,510 @@
...
+                "$ref": "/schema/fields/text"
+            },
+            "sameAs": [
+                "http://purl.org/dc/terms/alternative"
+            ],
+            "type": "array"
+        },
+        "bibliographicResourceType": {
+            "default": [],
+            "description": "The type of reference.",
```
Comment (cutoffthetop) [inline mex/model/entities/bibliographic-resource.json:52]:
```suggestion
            "description": "The type of bibliographic resource.",
```

File: mex/model/entities/bibliographic-resource.json [hunk 25]
```diff
@@ -0,0 +1,510 @@
...
+        "hadPrimarySource",
+        "identifier",
+        "identifierInPrimarySource",
+        "stableTargetId",
+        "title"
+    ],
+    "sameAs": [
+        "http://purl.org/dc/terms/BibliographicResource"
+    ],
+    "title": "BibliographicReference",
```
Comment (cutoffthetop) [inline mex/model/entities/bibliographic-resource.json:508]:
```suggestion
    "title": "BibliographicResource",
```

File: mex/model/entities/bibliographic-resource.json [hunk 26]
```diff
@@ -0,0 +1,510 @@
...
+            "description": "The range of pages or a single page."
+        },
+        "publicationYear": {
+            "anyOf": [
+                {
+                    "$comment": "year",
+                    "examples": [
+                        "2011",
+                        "1982",
+                        "1810"
```
Comment (cutoffthetop) [inline mex/model/entities/bibliographic-resource.json:346]:
```suggestion
                        "2024"
```

so it is inline with the other "year" field examples

File: mex/model/entities/bibliographic-resource.json [hunk 27]
```diff
@@ -0,0 +1,510 @@
...
+                    "type": "integer"
+                },
+                {
+                    "examples": [
+                        "Q3",
+                        "11/12",
+                        "Winter '23"
+                    ],
+                    "type": "string"
+                },
```
Comment (cutoffthetop) [inline mex/model/entities/bibliographic-resource.json:485]: this might be quite unstable, i'd rather just tread everything as a string (or null) then

```suggestion
                {
                    "examples": [
                        "2",
                        "Q3",
                        "11/12",
                        "Winter '23"
                    ],
                    "type": "string"
                },
```

File: mex/model/entities/bibliographic-resource.json [hunk 28]
```diff
@@ -0,0 +1,501 @@
...
+                    "type": "null"
+                }
+            ],
+            "default": null,
+            "description": "The volume of the periodical series."
+        }
+    },
+    "required": [
+        "accessRestriction",
+        "creator",
```
Comment (cutoffthetop) [inline mex/model/entities/bibliographic-resource.json:489]:
```suggestion
```

if `creator` has a `default: []`, then it shouldn't be listed as required

Comment (cutoffthetop) [reply mex/model/entities/bibliographic-resource.json:489]: or, i guess the other way round

File: mex/model/entities/bibliographic-resource.json [hunk 29]
```diff
@@ -0,0 +1,501 @@
...
+            "items": {
+                "$ref": "/schema/entities/organizational-unit#/identifier"
+            },
+            "subPropertyOf": [
+                "http://purl.org/dc/terms/contributor"
+            ],
+            "type": "array"
+        },
+        "creator": {
+            "default": [],
```
Comment (cutoffthetop) [inline mex/model/entities/bibliographic-resource.json:77]:
```suggestion
```

see https://github.com/robert-koch-institut/mex-model/pull/49/files#r1660814259

File: mex/model/entities/resource.json (hunk 16 above)
Comment (cutoffthetop) [reply mex/model/entities/resource.json:504]: re-opening this one, i still see `reference` here

File: README.md (hunk 1 above)
Comment (cutoffthetop) [inline README.md:42]:
```suggestion
```

we can remove this change from here, because it will be applied automatically by cruft and would only cause conflicts if it were here

File: mex/model/entities/bibliographic-resource.json [hunk 30]
```diff
@@ -0,0 +1,500 @@
...
+                    "examples": [
+                        "2011",
+                        "2019-03"
+                    ],
+                    "pattern": "^(?:\\d{4}|(?:\\d{4}-(?:0[1-9]|1[0-2])))$",
+                    "type": "string"
+                },
+                {
+                    "type": "null"
+                }
```
Comment (cutoffthetop) [inline mex/model/entities/bibliographic-resource.json:257]:
```suggestion
                {
                    "$comment": "year_month_day_time",
                    "examples": [
                        "2022-09-30T20:48:35Z"
                    ],
                    "format": "date-time",
                    "pattern": "^\\d{4}-(?:0[1-9]|1[0-2])-(?:0[1-9]|[12][0-9]|3[01])T(?:[0-1][0-9]|2[0-3]):[0-5][0-9]:[0-5][0-9]Z$",
                    "type": "string"
                },
                {
                    "$comment": "year_month_day",
                    "examples": [
                        "2014-08-24"
                    ],
                    "format": "date",
                    "pattern": "^\\d{4}-(?:0[1-9]|1[0-2])-(?:0[1-9]|[12][0-9]|3[01])$",
                    "type": "string"
                },
                {
                    "$comment": "year_month",
                    "examples": [
                        "2019-03"
                    ],
                    "format": "date",
                    "pattern": "\\d{4}-(?:0[1-9]|1[0-2])$",
                    "type": "string"
                },
                {
                    "$comment": "year",
                    "examples": [
                        "2024"
                    ],
                    "format": "date",
                    "pattern": "^\\d{4}$",
                    "type": "string"
                },
                {
                    "type": "null"
                }
```

File: mex/model/entities/bibliographic-resource.json [hunk 31]
```diff
@@ -0,0 +1,500 @@
...
+                    ],
+                    "type": "string"
+                },
+                {
+                    "type": "null"
+                }
+            ],
+            "default": null,
+            "description": "The name of the chapter of the publication, the book section belongs to."
+        },
```
Comment (cutoffthetop) [inline mex/model/entities/bibliographic-resource.json:409]:
```suggestion
        },
        "stableTargetId": {
            "$ref": "/schema/fields/identifier"
        },
```

File: mex/model/entities/variable.json [hunk 32]
```diff
@@ -26,30 +27,37 @@
...
+                        "image",
+                        "int55",
+                        "number"
                     ],
-                    "useScheme": "https://mex.rki.de/item/data-type"
+                    "type": "string"
                 },
                 {
-                    "type": "null"
+                    "type": null
```
Comment (cutoffthetop) [inline mex/model/entities/variable.json:49]:
```suggestion
                    "type": "null"
```

File: mex/model/entities/resource.json [hunk 33]
```diff
@@ -307,45 +416,92 @@
...
+                        "0",
+                        "18"
+                    ],
+                    "type": "integer"
+                },
+                {
+                    "type": "null"
+                }
+            ],
+            "description": "Specifies the minimum age of the population within the data collection, expressed in years."
```
Comment (cutoffthetop) [inline mex/model/entities/resource.json:433]:
```suggestion
            "default": null,
            "description": "Specifies the minimum age of the population within the data collection, expressed in years."
```

File: mex/model/entities/resource.json [hunk 34]
```diff
@@ -267,19 +351,42 @@
...
+        "maxTypicalAge": {
+            "$comment": "Source: Draft of HealthDCAT-AP published on 31st Jan 2024 (Milestone M6.2 Technical working group on the transition from existing metadata templates to HealthDCAT-AP).",
+            "anyOf": [
+                {
+                    "examples": [
+                        "99",
+                        "21"
+                    ],
+                    "type": "integer"
+                }
```
Comment (cutoffthetop) [inline mex/model/entities/resource.json:382]:
```suggestion
                },
                {
                    "type": "null"
                }
```

File: mex/model/vocabularies/bibliographic-resource-type.json [hunk 35]
```diff
@@ -0,0 +1,255 @@
...
+            "en": "Journal"
+        }
+    },
+    {
+        "altLabel": [
+            {
+                "de": "Wissenschaftlicher Artikel"
+            },
+            {
+                "de": "Paper"
```
Comment (cutoffthetop) [inline mex/model/vocabularies/bibliographic-resource-type.json:126]:
```suggestion
                "en": "Paper"
```

Comment (cutoffthetop) [inline mex/model/vocabularies/bibliographic-resource-type.json:126]:
```suggestion
                "de": "Wissenschaftlicher Artikel",
                "en": "Paper"
```

File: mex/model/vocabularies/bibliographic-resource-type.json [hunk 36]
```diff
@@ -0,0 +1,255 @@
...
+            }
+        ],
```
Comment (cutoffthetop) [inline mex/model/vocabularies/bibliographic-resource-type.json:128]:
```suggestion
        "altLabel": {
            "de": "Wissenschaftlicher Artikel",
            "en": "Paper"
        },
```

File: mex/model/entities/bibliographic-resource.json [hunk 37]
```diff
@@ -0,0 +1,515 @@
...
+                    "type": "integer"
+                },
+                {
+                    "examples": [
+                        "Q3",
+                        "11/12",
+                        "Winter '23"
+                    ],
+                    "type": "string"
+                },
```
Comment (cutoffthetop) [inline mex/model/entities/bibliographic-resource.json:474]:
```suggestion
                {
                    "examples": [
                        "2",
                        "Q3",
                        "11/12",
                        "Winter '23"
                    ],
                    "type": "string"
                },
```

File: mex/model/entities/bibliographic-resource.json [hunk 38]
```diff
@@ -0,0 +1,515 @@
...
```
Comment (cutoffthetop) [inline mex/model/entities/bibliographic-resource.json:221]:
```suggestion
                {
                    "examples": [
                        "2",
                        "Q3",
                        "11/12",
                        "Winter '23"
                    ],
                    "type": "string"
                },
```

File: mex/model/entities/variable.json [hunk 39]
```diff
@@ -26,30 +27,37 @@
                 }
             ],
             "default": null,
+            "description": "n established standard to which the described resource conforms ([DCT, 2020-01-20](http://dublincore.org/specifications/dublin-core/dcmi-terms/2020-01-20/)).",
```
Comment (LeipoldL) [inline mex/model/entities/variable.json:30]:
```suggestion
            "description": "An established standard to which the described resource conforms ([DCT, 2020-01-20](http://dublincore.org/specifications/dublin-core/dcmi-terms/2020-01-20/)).",
```

**Review by LeipoldL — CHANGES_REQUESTED** [review]
fixed some typos

**Review by cutoffthetop — DISMISSED** [review]

**Review by cutoffthetop — APPROVED** [review]

**Review by LeipoldL — APPROVED** [review]
changes approved
